java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2673,6 +2675,57 @@ public T touch(@NonNull String[] keys) {
         return getThis();
     }
 
+    /**
+     * Counts the number of set bits (population counting) in a string stored at <code>key</code>.
```

Comment:
any changes made above should be updated here too

Reply:
Thanks! Addressed in https://github.com/aws/glide-for-redis/commit/d6d52304d595ca9b35aed9e19024ffc9dc7772de